galsim/image.py

```diff
@@ -506,6 +506,15 @@ def getYMin(self): return self.image.getYMin()
     def getYMax(self): return self.image.getYMax()
     def getBounds(self): return self.image.getBounds()
 
+    def getOuterBounds(self):
+        """Get the bounds of the outer edge of the pixels.
+
+        Equivalent to galsim.BoundsD(im.xmin-0.5, im.xmax+0.5, im.ymin-0.5, im.ymax+0.5)
+
+        @returns a BoundsD instance
+        """
+        return galsim.BoundsD(self.xmin-0.5, self.xmin+0.5, self.ymin-0.5, self.ymin+0.5)
```

Comment:
I think the second ```xmin``` and second ```ymin``` should be ```xmax``` and ```ymax```?
(Also, need unit test for this, if I'm right that this is a bug.)